### PR TITLE
hummus

### PR DESCRIPTION
**Pull request by balerion**
this adds a hummus recipe!

### CODE REVIEW

File: docs/source/Recipes/Appetizers/Dips_Hummus.rst [hunk 1]
```diff
@@ -1,5 +1,64 @@
...
+   - fresh garlic
+   - salt
+   - pepper
+   - sumac
+   - olive oil
+
+.. procedure::
+
+   Chickpeas
+   -----------------
```
Comment (carlogico) [inline docs/source/Recipes/Appetizers/Dips_Hummus.rst]: I don't think sections in the procedure are appropriate. I'd rather have a single block with how to cook chickpeas and how to make hummus, and add the tahini as a separate recipe (and rest file) and link it from within the procedure.

The rest of the PR looks good

Comment (carlogico) [reply docs/source/Recipes/Appetizers/Dips_Hummus.rst]: (See https://github.com/carlogico/Recipes/pull/15/checks#step:4:110)

**Comment by balerion** [comment]
adds hummus, tahini, and baba ganoush recipes.

File: docs/source/Recipes/Appetizers/Dips_Baba_ganoush.rst [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+   - 25g yoghurt
+   - fresh garlic
+   - salt
+   - sumac
+   - pomegranate
+   - olive oil
+
+.. procedure::
+
+   Eggplants
```
Comment (carlogico) [inline docs/source/Recipes/Appetizers/Dips_Baba_ganoush.rst]: Sphinx still complains about section title. I'd do something like:

```
   For the eggplants:
   
```
for lines 25 and 26

File: docs/source/Recipes/Appetizers/Dips_Baba_ganoush.rst [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+      Roast eggplants in the oven. This is a little messy, so put them on something to gather the liquids which will come out.
+
+      The goal is to cook them in the oven as if they were cooking on coals, my favorite way. So use high temperatures, I use 200°C, and roast them for 45min without any protection from the oven's heating elements. The skin needs to turn black and carbonize in some places. Don't worry! the inside won't burn. This is how you get the typical smokiness in the baba ganoush.
+      
+      Remember, a couple holes will prevent eggplant explosions. Up to you whether you like the excitement.
+
+      Let them cool slightly, and then tear them open a little bit. Leave them to drip for 10 min or so in a strainer.
+
+
+   Baba Ganoush
```
Comment (carlogico) [inline docs/source/Recipes/Appetizers/Dips_Baba_ganoush.rst]: same here

File: docs/source/Recipes/Appetizers/Dips_Hummus.rst [hunk 4]
```diff
@@ -1,5 +1,58 @@
...
+      Cooking your own chickpeas really makes a difference. If you have the time, presoak them over night, or put them directly in the pressure cooker for 10min with water and lots of salt, aim for mediterranean seawater.
+
+      The actual cooking comes after. Drain them, rinse them, and put them in a pressure cooker, or if you have time in a pot (for me terracotta pots work really well with legumes). Remember to add some whole garlic cloves and laurel leaves (I use one of each). In a pressure cooker they will take 45 min, in a pot they will take 1h30min. Add salt only when they are done, and leave them be for a little while. You should be able to squish them between your fingers. Keep the cooking water!
+
+      If you have canned chickpeas, and not a lot of time, this is where you start.
+
+
+   Hummus
```
Comment (carlogico) [inline docs/source/Recipes/Appetizers/Dips_Hummus.rst]: same here

File: docs/source/Recipes/Appetizers/Tahini.rst [hunk 5]
```diff
@@ -0,0 +1,22 @@
+   .. index::
```
Comment (carlogico) [inline docs/source/Recipes/Appetizers/Tahini.rst]: no indent here

File: docs/source/Recipes/Appetizers/Dips_Baba_ganoush.rst (hunk 2 above)
Comment (carlogico) [reply docs/source/Recipes/Appetizers/Dips_Baba_ganoush.rst]: Or even 

```
   *For the eggplants*:
```

File: docs/source/Recipes/Appetizers/Dips_Hummus.rst [hunk 6]
```diff
@@ -1,5 +1,55 @@
 .. index::
    single: dips; hummus
 
-Hummus
+To make hummus
```
Comment (carlogico) [inline docs/source/Recipes/Appetizers/Dips_Hummus.rst]: Last quip: I'd jus have this be

```suggestion
Hummus
```

**Pull request merged**